scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -227,6 +223,19 @@
 
     private it.sauronsoftware.cron4j.Scheduler metricsMonitorScheduler;
 
+    /**
+     * Attributes used for the signal api
+     */
+    private SynchronizationInternal publicStore;
```

Comment:
`publicStore`is the right variable name? I don't like much

Reply:
I didn't name it, it was like that :)